spanner_orm/admin/column.py

```diff
@@ -14,12 +14,16 @@
 # limitations under the License.
 """Model for interacting with Spanner column schema table."""
 
-from typing import Type
+import re
+from typing import Type, Union
 
 from spanner_orm import error
 from spanner_orm import field
 from spanner_orm.admin import schema
 
+_string_pattern = r"^STRING\([0-9]+\)+$"
```

Comment:
`re.compile`?

Reply:
ahhh that's what you meant ... my bad ... updating.

Reply:
okay just updated/tested and still works as expected.